internal/application/application.go

```diff
@@ -1,11 +1,16 @@
 package application
 
-import "ai-codereview/internal/domain"
+import (
+	"ai-codereview/internal/domain"
+
+	"github.com/google/go-github/v61/github"
+)
 
```

Comment:
Code Review 2024-08-18 02:28:47 - New lines 1 to 5:

import (
	"ai-codereview/internal/domain"

	"github.com/google/go-github/v61/github"
)